ansible/build-react.yaml

```diff
@@ -11,6 +11,9 @@
       - name: Pre checks
         include: ./tasks/check_pr_exists.yaml
 
+      - name: Check if valid author
```

Comment:
I think we should have this earlier in the play and just fail fast. So lets exit the playbook if `not is_valid_author`. What do you think?

Reply:
you can utilize the `fail_when: not is_valid_author`

Reply:
Shouldn't we check if PR exists and then check for valid pr author? 
I will move the is_valid_author check up one step to check if should build step.

Reply:
oh good point!

Reply:
my bad :D